test/js/bun/util/password.test.ts

```diff
@@ -228,12 +228,13 @@ for (let algorithmValue of algorithms) {
           }
 
           async function runSlowTestWithOptions(algorithmLabel: any) {
-            const algorithm = { algorithm: algorithmLabel, timeCost: 5, memoryCost: 4 };
```

Comment:
let's avoid this breaking change for our users by patching the zig std lib to not throw in this case, as it did before

Reply:
good point